Assets/SEE/UI/SettingsMenu.cs

```diff
@@ -49,6 +51,31 @@ public class SettingsMenu : PlatformDependentComponent
         /// </summary>
         protected override void StartDesktop()
         {
+            string path = Application.dataPath + "/StreamingAssets/KeyBindings.json";
+            string allKeys = JsonConvert.SerializeObject(KeyBindings.GetKeyCodes());
+            // when the file exists, read the file, when it does not exist, write to the file, to make reading possible.
+            if (File.Exists(path) && File.ReadAllText(path) != null)
+            {
+                string jsonContent = File.ReadAllText(path);
```

Comment:
Why do you read the file twice? In line 57 and here again. That could be avoided.